util/dev.js

```diff
@@ -16,8 +16,9 @@ pkg.description += " (Development)";
 pkg.dependencies = devPkg.dependencies;
 
 // Remove scripts, dev deps, etc.
-pkg.scripts = {};
-pkg.devDependencies = {};
```

Comment:
@coopy -- This is probably a good opportunity to remove this file entirely and use `builder-support gen-dev` like `builder-react-component`